libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/CameraViewModel.kt

```diff
@@ -12,6 +16,10 @@ sealed class CameraAction {
 
 class CameraViewModel : UIViewModel<CameraState, CameraAction>(CameraState()) {
 
+    private val _cameraPadding =
+        MutableStateFlow(EdgeInsets(0.0, 0.0, 0.0, 0.0))
+    val cameraPadding: StateFlow<EdgeInsets> = _cameraPadding.asStateFlow()
```

Comment:
I separated `cameraPadding` from `CameraState` to avoid frequent state changes on each layout update.

Reply:
Done